SayTheirNames/Source/Controller/Home/Person/Views/PersonPhotoTableViewCell.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  PersonPhotoTableViewCell.swift
+//  Say Their Names
+//
+//  Created by Manase on 04/06/2020.
+//  Copyright © 2020 Franck-Stephane Ndame Mpouli. All rights reserved.
+//
+
+import UIKit
+
+extension UITableViewCell {
+    static func register(to tableView: UITableView, identifier: String) {
+        tableView.register(Self.self, forCellReuseIdentifier: identifier)
+    }
+}
+
+class PersonPhotoTableViewCell: UITableViewCell {
```

Comment:
Think it would be better if we could have this in a `UIView` instead. That way we can reuse the logic of creating a view with a blurred background.

Reply:
I just committed the change. Now we can reuse ImageWithBlurView.